README.md

```diff
@@ -27,4 +27,4 @@ For more detailed information, you can view our complete documentation at [docs.
 
 ## Purchases.framework v4
 You're viewing the documentation for RevenueCat iOS SDK version 3.
-For documentation on version 4, visit [the docs for RevenueCat iOS SDK version 4.](https://revenuecat-docs.netlify.app/documentation/Revenuecat)
+For documentation on version 4, visit [the docs for RevenueCat iOS SDK version 4.](https://revenuecat.github.io/purchases-ios-docs/4.4.0/documentation/revenuecat/)
```

Comment:
Let's just have it point to the root, which should redirect to the latest version automatically:

```suggestion
For documentation on version 4, visit [the docs for RevenueCat iOS SDK version 4.](https://revenuecat.github.io/purchases-ios-docs/)
```